infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoService.kt

```diff
@@ -250,6 +250,8 @@ constructor(
 
     private fun pushChanges(layoutBranch: LayoutBranch, publications: List<PublicationDetails>) {
         val ratkoPushId = ratkoPushDao.startPushing(publications.map { it.id })
+        logger.info("Starting ratko push id=$ratkoPushId")
```

Comment:
Tällaiset ovat lokin tulkinnan kannalta mun mielestä mielekkäitä, ns. antaa suoran refen kantariviin minkä lokit kyseessä.